amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/model/behaviour/AbstractDisposableItem.java

```diff
@@ -86,4 +113,39 @@ protected void cleanDisposableItem(NodeService nodeService, NodeRef nodeRef)
         }
     }
 
+    /**
+     * Cleans and re-initiates the containing records
+     *
+     * @param childAssociationRef
+     */
+    protected void reinitializeRecordFolder(ChildAssociationRef childAssociationRef)
+    {
+
+        NodeRef newNodeRef = childAssociationRef.getChildRef();
+
+        AuthenticationUtil.runAs(new AuthenticationUtil.RunAsWork<Object>()
```

Comment:
You can replace the `new AuthenticationUtil.RunAsWork<Object>()...` with a shorter _lambda_ here (as suggested by the IDE/IntelliJ).

Reply:
Will add that in the next commit, thank you.